FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/include/plaintext_freertos.h

```diff
@@ -61,6 +58,16 @@ struct NetworkContext
     Socket_t tcpSocket;
 };
 
+/**
+ * @brief Plain text transport Connect / Disconnect return status.
+ */
+typedef enum PlaintextTransportStatus
+{
+    PLAINTEXT_TRANSPORT_INVALID_PARAMETER = 0, /**< At least one parameter was invalid. */
+    PLAINTEXT_TRANSPORT_SUCCESS = 1,           /**< Function successfully completed. */
+    PLAINTEXT_TRANSPORT_CONNECT_FAILURE        /**< Initial connection to the server failed. */
+} PlaintextTransportStatus_t;
```

Comment:
(Nit) I would say that the same order can be maintained when starting with `1` for success
```suggestion
typedef enum PlaintextTransportStatus
{
    PLAINTEXT_TRANSPORT_SUCCESS = 1,           /**< Function successfully completed. */
    PLAINTEXT_TRANSPORT_INVALID_PARAMETER = 2, /**< At least one parameter was invalid. */
    PLAINTEXT_TRANSPORT_CONNECT_FAILURE = 3    /**< Initial connection to the server failed. */
} PlaintextTransportStatus_t;
```